cfgov/v1/tests/test_links.py

```diff
@@ -16,19 +16,19 @@ def test_external_link(self):
         link = '<a href="https://wwww.google.com">external link/a>'
         output = str(parse_links(link))
         self.assertIn('external-site', output)
-        self.assertIn('class="icon-link_text"', output)
+        self.assertIn('cf-icon-external-link', output)
 
     def test_cfpb_link(self):
         link = '<a href="http://www.consumerfinance.gov/foo">cfpb link</a>'
         output = str(parse_links(link))
         self.assertNotIn('external-site', output)
-        self.assertNotIn('class="icon-link_text"', output)
+        self.assertNotIn('cf-icon-external-link', output)
 
     def test_gov_link(self):
         link = '<a href="http://www.fdic.gov/bar">gov link</a>'
         output = str(parse_links(link))
         self.assertNotIn('external-site', output)
-        self.assertIn('class="icon-link_text"', output)
+        self.assertIn('cf-icon-external-link', output)
```

Comment:
Is there a compelling reason to remove the `class=""` portion of the assertion? Can/will the class not only be `cf-icon-external-link`?